packages/accounts/wagmi.config.ts

```diff
@@ -20,6 +20,11 @@ export default defineConfig(
         address: config.address,
       },
     ],
-    plugins: [plugingen({ chain: config.chain, rpcUrl: config.rpcUrl })],
+    plugins: [
+      plugingen({
+        chain: config.chain,
+        connectionConfig: { rpcUrl: config.rpcUrl! },
```

Comment:
isn't rpcUrl optional?


```suggestion
        connectionConfig: { rpcUrl: config.rpcUrl },
```

Reply:
no, it's using ConnectionConfig like in AlchemyProvider

Reply:
yea but here https://github.com/alchemyplatform/aa-sdk/pull/314/files#diff-29103509588947d27bb0a950af40aa9d8eace89db6ef63d3927cee22e466dab7R31-R36 you still have to check for this null case directly

Reply:
is `!` just something javascript ignores? we don't set `config.rpcUrl` anywhere in our plugin configs. I think here is where should handle that case with something like:

```suggestion
        connectionConfig: config.rpcUrl ? { rpcUrl: config.rpcUrl } : {apiKey: process.env.API_KEY },
```